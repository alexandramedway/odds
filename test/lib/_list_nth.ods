do nums = [1, 2, 3, 4, 5]
do print(list_nth(2, l))
do print(list_nth(5, l))
