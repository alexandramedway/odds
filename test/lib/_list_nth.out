3
list_nth: List index out of range
